pkg/kt/snmp.go

```diff
@@ -237,14 +237,26 @@ type SnmpDeviceMetric struct {
 }
 
 const (
-	SNMP_GOOD = 1
-	SNMP_BAD  = 2
+	SNMP_GOOD                   = 1
+	SNMP_BAD_NO_DATA            = 3
+	SNMP_BAD_DEVICE_ERR         = 4
+	SNMP_BAD_INTERFACE_ERR      = 5
+	SNMP_BAD_POLL_TIMEOUT       = 6
+	SNMP_BAD_INIT_METADATA      = 7
+	SNMP_BAD_INIT_METRICS       = 8
+	SNMP_BAD_FIRST_METRICS_POLL = 9
```

Comment:
nit: is there a reason `2` was skipped? This goes 1 -> 3,4,5...9

Reply:
Old pattern was just 

```
const (
	SNMP_GOOD = 1
	SNMP_BAD  = 2
)
```

Taking out SNMP_BAD and I didn't want to confuse old data. Could keep things 1,2,3... though as this is kinda weird. Thoughts?

Reply:
Oh hmm, the way I read this was that the actual data set in `kentik.snmp.PollingHealth` came from `SNMP_STATUS_MAP`, so the actual values were only ever `GOOD`|`BAD`. The numbers are just to represent the enum, right? Or am I reading this wrong 🥴

Reply:
That's the old way. The new way (which is totally a hack) still gets the string values from SNMP_STATUS_MAP but splits the string into 2 parts, one which is `GOOD|BAD` and the other is the reason. But the int part which the guadge value is set to is either 1 or 3-9 depending on the actual reason.

Reply:
>But the int part which the guadge value is set to is either 1 or 3-9 depending on the actual reason.
Ah, I didn't realize the gauge is set from this. Makes sense when you put it that way 👍🏻

Reply:
I was gonna try and keep it 1 or 2 but then it seemed easy enough to alert on 1 or not 1.